packages/react-component-library/src/components/Dialog/Dialog.test.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import '@testing-library/jest-dom/extend-expect'
+import { render, RenderResult, fireEvent } from '@testing-library/react'
+
+import { Dialog } from './Dialog'
+
+describe('Modal', () => {
+  let wrapper: RenderResult
+  let title: string
+  let description: string
+  let onConfirm: (event: any) => void
```

Comment:
consider using jest.fn()

Reply:
For declaring the type? I am using jest.fn() for mocking the function.

Reply:
for onConfirm

Reply:
https://github.com/Royal-Navy/standards-toolkit/pull/290/files/78f5a0fa997414087031540ecf35a93cda02c0b2#diff-dc6c405941c0cf8053bcd157087bb201R19